modules/reindex/src/main/java/org/opensearch/index/reindex/remote/RemoteScrollableHitSource.java

```diff
@@ -186,8 +186,9 @@ public void onSuccess(org.opensearch.client.Response response) {
                             }
                             if (xContentType == null) {
                                 try {
+                                    logger.debug("Response didn't include Content-Type: " + bodyMessage(response.getEntity()));
```

Comment:
Do we need this log here?

Reply:
Yes. The response can be used for debugging purpose to check what is the actual response from the endpoint.